### PR TITLE
Add a check about optional column in converter

### PR DESCRIPTION
**Pull request by Alonreznik**
* Fix - trying to fix the "default already set" in SchemaBuilder error
* Enhancement - Add extra logs to column on registration

### CODE REVIEW

File: src/main/java/oryanmoshe/kafka/connect/util/TimestampConverter.java [hunk 1]
```diff
@@ -66,13 +66,16 @@ public void configure(Properties props) {
     public void converterFor(RelationalColumn column, ConverterRegistration<SchemaBuilder> registration) {
         if (this.debug)
             System.out.printf(
-                    "[TimestampConverter.converterFor] Starting to register column. column.name: %s, column.typeName: %s%n",
-                    column.name(), column.typeName());
+                    "[TimestampConverter.converterFor] Starting to register column. column.name: %s, column.typeName: %s, column.hasDefaultValue: %s, column.defaultValue: %s column.isOptional: %s%n",
```
Comment (oryanmoshe) [inline src/main/java/oryanmoshe/kafka/connect/util/TimestampConverter.java:69]:
```suggestion
                    "[TimestampConverter.converterFor] Starting to register column. column.name: %s, column.typeName: %s, column.hasDefaultValue: %s, column.defaultValue: %s, column.isOptional: %s%n",
```

File: src/test/java/oryanmoshe/kafka/connect/util/testMe.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package oryanmoshe.kafka.connect.util;
+
+public class testMe {
+
+}
```
Comment (oryanmoshe) [inline src/test/java/oryanmoshe/kafka/connect/util/testMe.java:5]: Delete file

File: src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java [hunk 3]
```diff
@@ -88,6 +88,7 @@ void converterTest(final String columnType, final String format, final String in
                     String.format("format.%s",
                             columnType.equals("timestamp") || columnType.equals("datetime2") ? "datetime" : columnType),
                     format);
+        props.put("debug", "true");
```
Comment (oryanmoshe) [inline src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java:91]: Needed?

**Review by oryanmoshe — APPROVED** [review]

File: src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java (hunk 3 above)
Comment (Alonreznik) [reply src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java:91]: Yes. I want to make sure the debug is turning on in Testing, for better visibility.

**Pull request merged**